ivy/functional/frontends/tensorflow/math.py

```diff
@@ -0,0 +1,16 @@
+# global
+import ivy
+
+
+def add(x, y, name=None):
```

Comment:
this is also in layers? Is this a mistake, or it appears in both categories?

Reply:
Yes, It appears in both categories. I've also attached the link of TensorFlow API of layers and math below just for reference.
Layers: https://www.tensorflow.org/api_docs/python/tf/keras/layers#functions_2
Math: https://www.tensorflow.org/api_docs/python/tf/math#functions